### ISSUE TITLE
IPv6 prefix filter generation empty for route-set object

### ISSUE THREAD

**Issue by pierreb4** [snar/bgpq3#50]
This is broken in bgpq3-0.1.31

Expected output:

```$ bgpq3 -T -6 RS-NETNOD-ANYCAST
no ipv6 prefix-list NN
ipv6 prefix-list NN permit 2001:67c:1010::/47
ipv6 prefix-list NN permit 2001:67c:1010::/48
ipv6 prefix-list NN permit 2001:67c:1011::/48
ipv6 prefix-list NN permit 2001:7fe::/32
ipv6 prefix-list NN permit 2001:7fe::/33
ipv6 prefix-list NN permit 2001:dd8:10::/47
ipv6 prefix-list NN permit 2001:dd8:10::/48
ipv6 prefix-list NN permit 2001:dd8:11::/48
ipv6 prefix-list NN permit 2a01:3f1::/38
ipv6 prefix-list NN permit 2a01:3f1:3000::/38
ipv6 prefix-list NN permit 2a01:3f1:5000::/38
ipv6 prefix-list NN permit 2a01:3f1:8000::/38
ipv6 prefix-list NN permit 2a01:3f1:a000::/38
ipv6 prefix-list NN permit 2a01:3f1:c000::/38
```

Current output:

```$ bgpq3 -6 RS-NETNOD-ANYCAST
no ipv6 prefix-list NN
! generated prefix-list NN is empty
ipv6 prefix-list NN deny ::/0
```

Using the same function call on line 690 as the one on line 687 fixes it for this object (with pipeline disabled). It might break other things though.

Thanks!

**Comment by snar** [comment]
On Tue, Mar 12, 2019 at 10:09:52AM +0000, pierreb4 wrote:
> This is broken in bgpq3-0.1.31

Looks like it is already fixed in 0.1.35 and HEAD. Can I ask
you to update and check again ?

**Comment by pierreb4** [comment]
Ah yes, this works with bgpq3-0.1.35.

I grabbed 0.1.31 because it looked like the newest at: http://snar.spb.ru/prog/bgpq3/

Closing this. Thanks again!
